terraform/deployments/ga4-analytics/project_iam_binding.tf

```
@@ -0,0 +1,75 @@
+resource "google_project_iam_binding" "project-owners" {
+  project = google_project.project.project_id
+  role    = "roles/owner"
+
+  members = [
+    "group:gcp-ga4-analytics-352613-owners@digital.cabinet-office.gov.uk",
+    "serviceAccount:terraform-cloud-production@govuk-production.iam.gserviceaccount.com"
+  ]
+}
+
+resource "google_project_iam_binding" "project-editors" {
+  project = google_project.project.project_id
+  role    = "roles/editor"
+  members = [
+    "group:gcp-ga4-analytics-352613-editors@digital.cabinet-office.gov.uk",
+  ]
+}
+
+resource "google_project_iam_binding" "project-viewers" {
+  project = google_project.project.project_id
+  role    = "roles/viewer"
+  members = [
+    # purposfully empty
+  ]
+}
+
+resource "google_project_iam_binding" "project-gds_bigquery_editor" {
+  project = google_project.project.project_id
+  role    = "projects/ga4-analytics-352613/roles/GDS_BQ_editor"
```

For these bindings use references to the custom roles name, rather than hard coding 
```suggestion
  role    = google_project_iam_custom_role.roles--GDS_BQ_editor.name
```

---

thanks, good change